src/components/hoc/homeLayout/HomeLayout.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Layout, Typography } from 'antd';
+import { FOOTER_TEXT } from './constants';
+import Logo from '../../../images/neoNatar Logo.svg';
+
+const { Content, Header, Footer } = Layout;
+const { Text } = Typography;
+const HomeLayout = props => (
+    <Layout>
+        <Header className="header">
+            <img src={Logo} width="120px" height="60px" alt="NeoNatar Logo" />
+        </Header>
+        <Content className="content">{props.children}</Content>
+        <Footer className="footer text-center">
+            <Typography>
+                <Text strong>
+                    {FOOTER_TEXT}
+                </Text>
+            </Typography>
+        </Footer>
+    </Layout>
+);
+export default HomeLayout;
```

Comment:
I have not tested this on-screen yet to know exactly what this achieves but if you are trying to build this for App level I think you should separate the Header and Footer in different modules. For the header you may want to edit from the existing one.

Reply:
Alright. Would do that now